benchmark_runner/workloads/bootstorm_vm.py

```diff
@@ -196,7 +196,9 @@ def _verify_vm_ssh(self):
                             failure = True
                         upgrade_done = self._oc.get_cluster_status() == f'Cluster version is {self._wait_for_upgrade_version}'
 
-                    current_wait_time += self._oc.SHORT_TIMEOUT # Increment the wait time
+                    # Sleep 1 sec between each cycle
+                    time.sleep(1)
+                    current_wait_time += 1  # Increment the wait time
```

Comment:
Ideally you'd check the actual time rather than just incrementing the current_wait_time, but for this purpose where the exact timeout doesn't matter, this is OK.